docs/docusaurus/docs/terms/batch.md

```diff
@@ -27,7 +27,7 @@ Batches are also used by <TechnicalTag relative="../" tag="validator" text="Vali
 
 When creating Expectations interactively, a <TechnicalTag relative="../" tag="validator" text="Validator" /> needs access to a specific Batch of data against which to check Expectations. The [how to guide on interactively creating expectations](../guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md) covers using a Batch in this use case.
 
-To specify which Batches of data should be used when using Great Expectations to generate statistics and candidate Expectations for your data. see [How to create and edit Expectations with the User Configurable Profiler](../guides/expectations/how_to_create_and_edit_expectations_with_a_profiler.md).
+To specify which Batches of data should be used when using Great Expectations to generate statistics and candidate Expectations for your data.
```

Comment:
```suggestion
```
This entire section can be removed.  Without the link it's a sentence fragment and serves no purpose.

Reply:
@Rachel-Reverie This seems like it was already done.  Thanks.